AutomatedTesting/Gem/PythonTests/Terrain/EditorScripts/Terrain_World_ConfigurationWorks.py

```diff
@@ -0,0 +1,163 @@
+        editor.EditorComponentAPIBus(bus.Broadcast, "SetComponentProperty", terrain_world_component, "Configuration|World Bounds (Min)", world_bounds_min)
+        editor.EditorComponentAPIBus(bus.Broadcast, "SetComponentProperty", terrain_world_component, "Configuration|Height Query Resolution (m)", height_query_resolution)
+        world_max = hydra.get_component_property_value(terrain_world_component, "Configuration|World Bounds (Max)")
+        world_min = hydra.get_component_property_value(terrain_world_component, "Configuration|World Bounds (Min)")
+        world_query = hydra.get_component_property_value(terrain_world_component, "Configuration|Height Query Resolution (m)")
+        Report.result(Tests.bounds_max_changed, world_max == world_bounds_max)
+        Report.result(Tests.bounds_min_changed, world_min == world_bounds_min)
+        Report.result(Tests.height_query_changed, world_query == height_query_resolution)
+
+        # 6) Change the Axis Aligned Box Shape dimensions
+        box_dimensions = azmath.Vector3(SET_BOX_X_SIZE, SET_BOX_Y_SIZE, SET_BOX_Z_SIZE)
+        terrain_spawner_entity.get_set_test(0, "Axis Aligned Box Shape|Box Configuration|Dimensions", box_dimensions)
+        box_shape_dimensions = hydra.get_component_property_value(terrain_spawner_entity.components[0], "Axis Aligned Box Shape|Box Configuration|Dimensions")
+        Report.result(Tests.box_dimensions_changed, box_dimensions == box_shape_dimensions)
+        
+        # 7) Set the Vegetation Shape reference to terrain_spawner_entity
+        height_provider_entity.get_set_test(0, "Configuration|Shape Entity Id", terrain_spawner_entity.id)
+        entityId = hydra.get_component_property_value(height_provider_entity.components[0], "Configuration|Shape Entity Id")
+        Report.result(Tests.shape_changed, entityId == terrain_spawner_entity.id)
+
+        # 8) Set the FastNoise Gradient frequency to 0.01
+        Frequency = 0.01
+        height_provider_entity.get_set_test(2, "Configuration|Frequency", Frequency)
+        FrequencyVal = hydra.get_component_property_value(height_provider_entity.components[2], "Configuration|Frequency")
+        Report.result(Tests.frequency_changed, math.isclose(Frequency, FrequencyVal, abs_tol = 0.00001))
+
+        # 9) Set the Gradient List to height_provider_entity
+        propertyTree = hydra.get_property_tree(terrain_spawner_entity.components[2])
+        propertyTree.add_container_item("Configuration|Gradient Entities", 0, height_provider_entity.id)
+        checkID = propertyTree.get_container_item("Configuration|Gradient Entities", 0)
+        Report.result(Tests.entity_added, checkID.GetValue() == height_provider_entity.id)
+
+        # 10) Disable and Enable the Terrain Gradient List so that it is recognised
+        editor.EditorComponentAPIBus(bus.Broadcast, 'EnableComponents', [terrain_spawner_entity.components[2]])
```

Comment:
comment indicates disable and enable, but the action is one call to EnableComponents. is that the same as disable and enable?

Reply:
Yes, changed comment to clarify.